### PR TITLE
Tutorial adjustments

### PR DESCRIPTION
**Pull request by mosampaio**

### CODE REVIEW

File: tutorial/index.jade [hunk 1]
```diff
@@ -1,105 +1,86 @@
...
 
-      1. The users begin the registration process by entering their data,
-         including a phone number into a signup form.
-      2. The authentication system sends a one-time password to the user's
-         mobile phone to verify the possession of that phone number.
-      3. The user enters the one-time password into a form before completing
-         registration.
-      4. The user sees a success page and receives an SMS indicating that
-         his/her account has been create.
+      1. The users begin the registration process by entering their data, including a phone number, into a signup form
```
Comment (acamino) [inline tutorial/index.jade]: Should we use a period at the end of each sentence? Please take a look at http://www.mhhe.com/mayfieldpub/tsw/period.htm

File: tutorial/index.jade [hunk 2]
```diff
@@ -1,105 +1,86 @@
...
 
-  .step(
-    data-title='User Model',
-    data-file='src/main/java/com/twilio/verification/model/User.java')
+  .step(data-title='User Model', data-file='src/main/java/com/twilio/verification/model/User.java', data-highlight='7-37')
     :markdown
       ## The User Model
 
-      The User Model for this use-case is pretty straightforward and JPA
+      The User Model for this use-case is pretty straight-forward and JPA
```
Comment (acamino) [inline tutorial/index.jade]: Doesn't seem right the use of _straight-forward_. Please take a look at http://www.merriam-webster.com/dictionary/straightforward.

File: tutorial/index.jade [hunk 3]
```diff
@@ -108,88 +89,82 @@
...
-
-      Then all we need to do to is send an SMS to the user's phone. This is done
-      using the `send()` method. Notice that we'll use the user's `fullNumber()` to
-      make sure we support international numbers. The `getFullPhoneNumber()` method
-      in the `User` model simply returns a combination of the `countryCode` and
-      `phoneNumber` that the user provided upon registration.
+      We create a single instance of the Twilio REST API helper,
+      called `client` in this example.
 
+      Then we need get the account, get the messageFactory and finally use its `sendMessage` method
```
Comment (acamino) [inline tutorial/index.jade]: Would be better if we use something like "..., get the `messageFactory` method and ...". What do you think?

Comment (mosampaio) [inline tutorial/index.jade]: agreed.

File: tutorial/index.jade (hunk 1 above)
Comment (mosampaio) [inline tutorial/index.jade]: Makes sense.

**Comment by acamino** [comment]
Thanks @mosampaio! LGTM.

**Pull request merged**